.buildkite/commands/build.sh

```diff
@@ -1,9 +1,6 @@
 #!/bin/bash -u
 
 echo "--- Set up SPM"
-# We'd like to use this, but it doesn't work yet. See
-# https://github.com/Automattic/bash-cache-buildkite-plugin/issues/20 ...
-#
```

Comment:
This comment was a leftover. But looking at the rest of the code it seems like all the following SSH code is unnecessary now that 1) everything is private and 2) we have a CI pluging command for SPM